### DISCUSSION TITLE
Is Deta Space Free and How many mb storge do we have

### DISCUSSION THREAD

**Discussion by HakanOZDABAK** [deta/space-docs#78] [Q&A]
Hello everyone. I wanna ask something.

1-)Is Data Space domain storage and database side free? I published my backend project and I connected with my own domain but I am curious that how many project and domain can be free in one account?

2-)My last question is about Collections. Can we publish our SQL database or it has own different connection type and can we use our on backend project? Also how many mb do we have for storage in Collections.

Thanks for helping.

**Comment by sudo-subg** [comment]
> Is Data Space domain storage and database side free? I published my backend project and I connected with my own domain but I am curious that how many project and domain can be free in one account?

Deta Drive has a limit of 10 GB per account. Micros have a limit of 250 MB including the final deployment package with deps and all. As of now there are no limits for Deta Base. And there is no limit on number of projects created and linked with custom domains.

> My last question is about Collections. Can we publish our SQL database or it has own different connection type and can we use our on backend project? Also how many mb do we have for storage in Collections.

If you are asking about using a SQL database with Space micros, I can't confirm if it will work or not. But as Deta Space is a serverless platform, micros will re-establish the connection each time they will be invoked. And Collection doesn't have any separate limitation, but the Base and Drive instances in it have the same limitations as usual.

**Reply by Legend-Mukund** [reply]
Just 10gb drive storage

**Comment by datasquad-ai** [comment]
How can we increase the limit of 250 MB per micro?
